src/js/Content/Features/Community/FConfirmDeleteComment.js

```diff
@@ -0,0 +1,61 @@
+import {Localization, SyncedStorage} from "../../../modulesCore";
+import {Feature, Messenger, User} from "../../modulesContent";
+import {Page} from "../Page";
+
+export default class FConfirmDeleteComment extends Feature {
+
+    checkPrerequisites() {
+        return User.isSignedIn && SyncedStorage.get("confirmdeletecomment");
+    }
+
+    apply() {
+
+        Messenger.addMessageListener("setOption", value => SyncedStorage.set("confirmdeletecomment", value));
```

Comment:
This feature is enabled on all community-based pages, so is it ok to use such a generic name? What happens if another feature adds a listener with the same name?